src/ImageSharp/Formats/Jpeg/Components/Encoder/HuffmanScanEncoder.cs

```diff
@@ -11,6 +15,12 @@ namespace SixLabors.ImageSharp.Formats.Jpeg.Components.Encoder
 {
     internal class HuffmanScanEncoder
     {
+        /// <summary>
+        /// Compiled huffman tree to encode given values.
+        /// </summary>
+        /// <remarks>Yields codewords by index consisting of [run length | bitsize].</remarks>
+        private HuffmanLut[] huffmanTables;
```

Comment:
Note: This doesn't really do anything performance-wise but makes it easier to implement image specific huffman coding tree in the future without changing anything in the underlying stream writing logic.